tests/benchmarks/CompiledCodeBenchmarks/MicroPerf/Benchmarks.fs

```diff
@@ -1,11 +1,16 @@
-namespace MicroPerf
+module MicroPerf.Program
```

Comment:
Maybe you can also rename the file to `Program` then.

Reply:
02062e9